edi_oca/models/edi_exchange_record.py

```diff
@@ -108,12 +109,11 @@ class EDIExchangeRecord(models.Model):
         ),
     ]
 
-    @api.depends("type_id.code", "model", "res_id")
-    def _compute_name(self):
+    @api.depends("model", "res_id")
```

Comment:
```suggestion
    @api.depends("record.display_name")
```
Afais previously instead of an empty name we set to default "Unrelated" value, but now when it is empty we can use directly related field

Reply:
you cannot depend on `record`. It's a property. We should have a Reference field but I don't like to store the same info twice. Also, I removed "Unrelated" because: it made poor sense overall and no sense at all when the exchange does not touch any real record -> no relation needed.
When you have a relation, you already see that on the form.